tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js

```diff
@@ -0,0 +1,98 @@
+// <reference types="Cypress" />
+
+describe('Step Ecommerce Tax Information', function () {
+    before(() => {
+        cy.setCustomerData();
+        cy.visit(
+            'wp-admin/?page=nfd-onboarding&flow=ecommerce#/ecommerce/step/tax'
+        );
+        // cy.injectAxe();
+        cy.wait(2000);
+    });
+
+    // it('Is Accessible', () => {
+	// 	cy.wait(500);
+	// 	cy.checkA11y();
+	// });
+
+    it('Checks if Drawer opened', () => {
+        cy.get('.nfd-onboarding-drawer__panel-inner')
+            .scrollIntoView()
+            .should('be.visible');
+    });
```

Comment:
Can we add a testcase to check if the respective page is selected and highlighted in the `Drawer Panel` ? As I remember David mentioning that we should check that too
![image](https://user-images.githubusercontent.com/53893563/187179002-98726912-521f-46c6-af99-570beaad09b0.png)

Reply:
Line 24-29 is what you missed looking at xD

Reply:
Although there is a typo I'll fix that in the test case name

Reply:
Fixed typo

Reply:
great, thanks! Rest all looks good